### ISSUE TITLE
Path must be a string

### ISSUE THREAD

**Issue by kadiks** [facebookincubator/exerslide#7]
Since the update, I can't get any presentation to build or serve.

I used the following commands:

```
npm i exerslide-cli -g
mkdir test3
cd test3
exerslide init test3
exerslide build
```

On node 6 and 5.
I also installed it on another computer and I still get the same error:

```
Generating presentation in "out" ...
Presentation generation failed!
List of errors:
path.js:8
    throw new TypeError('Path must be a string. Received ' +
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.relative (path.js:496:3)
    at formatWebpackError (/Users/[...]/tutorial/test3/node_modules/exerslide/lib/builder.js:369:29)
    at /Users/[...]/tutorial/test3/node_modules/exerslide/lib/builder.js:343:33
    at Array.forEach (native)
    at logWebpackErrors (/Users/[...]/tutorial/test3/node_modules/exerslide/lib/builder.js:342:30)
    at /Users/[...]/tutorial/test3/node_modules/exerslide/lib/builder.js:194:9
    at Compiler.<anonymous> (/Users/[...]/tutorial/test3/node_modules/webpack/lib/Compiler.js:194:14)
    at Compiler.emitRecords (/Users/[...]/tutorial/test3/node_modules/webpack/lib/Compiler.js:282:37)
    at Compiler.<anonymous> (/Users/[...]/tutorial/test3/node_modules/webpack/lib/Compiler.js:187:11)
    at /Users/[...]/tutorial/test3/node_modules/webpack/lib/Compiler.js:275:11
    at Compiler.applyPluginsAsync (/Users/[...]/tutorial/test3/node_modules/tapable/lib/Tapable.js:60:69)
    at Compiler.afterEmit (/Users/[...]/tutorial/test3/node_modules/webpack/lib/Compiler.js:272:8)
    at Compiler.<anonymous> (/Users/[...]/tutorial/test3/node_modules/webpack/lib/Compiler.js:267:14)
    at /Users/[...]/tutorial/test3/node_modules/async/lib/async.js:52:16
    at done (/Users/[...]/tutorial/test3/node_modules/async/lib/async.js:246:17)
```

Any idea where that could come from?
Until then, I'll revert to the previous version.

Thanks

**Comment by tmair** [comment]
I've got the same error and did some debugging. The first results are:
- There is a module not found error: `"Module not found: Error: Cannot resolve 'file' or 'directory' \slides\css\.css inslides"`
- The `error.module.resource` property is not defined on https://github.com/facebookincubator/exerslide/blob/f19909cfd42a2d66ddf546d8e5ba89a36359bef3/packages/exerslide/lib/builder.js#L369-L372 

Therefore an exception is thrown because `undefined` is passed into `path.resolve`

**Comment by fkling** [comment]
Yep, that is the same issues as #6 . I have a fix for this, will release it very soon. Sorry for the inconvenience!

**Comment by fkling** [comment]
This is fixed with via 84c6aec, bcc9c00, c839be2 and 986e07b.

v1.1.2 was published.
